modules/_labs/color-picker/react/lib/parts/ColorReset.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import {borderRadius, colors, spacing, type, inputColors} from '@workday/canvas-kit-react-core';
+
+import {Swatch} from './Swatch';
+
+export const ResetButton: React.FunctionComponent<Props> = ({id, onClick, resetColor, label}) => {
+  const handleResetColor = () => onClick(resetColor);
+
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Enter') {
+      handleResetColor();
+    }
+  };
```

Comment:
Did you check if this was actually necessary? It is making your `onClick` type signature more difficult than it needs to be and `<button>` elements already file the `onClick` event when they are focused and you hit the `Enter` or `Space` keys. I suggest removing this.